book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] @momo-87 Great work on your book class however you forgot to add the `add_rental` method that takes parameters person date and the book itself in order to achieve has many association between book and rental.